packages/webdriverio/src/commands/element/click.ts

```diff
@@ -1,7 +1,8 @@
 import { getBrowserObject } from '../../utils/index.js'
 import type { ClickOptions } from '../../types.js'
 import type { Button } from '../../utils/actions/index.js'
-
+import logger from '@wdio/logger'
```

Comment:
While we don't have an explicit import order defined as an eslint rule let's still have dependency imports first, then an empty line and then module imports.